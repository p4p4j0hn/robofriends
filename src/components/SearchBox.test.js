import React from 'react';
import { shallow } from 'enzyme';
import SearchBox from './SearchBox';

describe('searchbox', () => {
  it('expect to render SearchBox component', () => {
    expect.assertions(1);
    expect(shallow(<SearchBox />)).toMatchSnapshot();
  });
});
