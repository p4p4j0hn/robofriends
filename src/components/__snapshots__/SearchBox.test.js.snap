// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`searchbox expect to render SearchBox component 1`] = `
<div
  className="pa2"
>
  <input
    aria-label="Search Robots"
    className="pa3 ba b--red bg-lightest-red"
    name="search robot list"
    placeholder="search robots"
    type="search"
  />
</div>
`;
